src/main/java/pages/HomePage.java

```diff
@@ -31,55 +32,76 @@
     @FindBy(linkText = "Edit Profile Settings")
     private WebElement lnkEditProfileSettings;
 
+    @FindBy(css = ".article-preview:nth-child(1) .ion-heart")
+    private WebElement heartIcon;
```

Comment:
You are always clicking the first element you find. This might be more dynamic if you always choose a random one. Take this as a suggestion

Reply:
I understand, for this test, I left it like this with the first item. But future could be adjusted to randomly select one